src/integrationTest/java/uk/gov/hmcts/darts/common/controller/CourthouseApiTest.java

```diff
@@ -144,6 +153,39 @@ void courthousesWithRegionAndSecurityGroupsGet() throws Exception {
 
     }
 
+    @Test
+    @Transactional
+    void courthousesGetWithRegionId() throws Exception {
+        CourthouseEntity ch1 = dartsDatabase.createCourthouseUnlessExists("HAVERFORDWEST");
+
+        RegionEntity region1 = new RegionEntity();
+        region1.setRegionName("Reading");
+        regionRepository.save(region1);
+        ch1.setRegion(region1);
+
+        dartsDatabase.createCourthouseUnlessExists("SWANSEA");
+
+        CourthouseEntity ch3 = dartsDatabase.createCourthouseUnlessExists("MANCHESTER");
+        RegionEntity region3 = new RegionEntity();
+        region3.setRegionName("KNOWLE");
```

Comment:
Not that it really matters, but suggest using a region name more aligned with the region names we'll actually use. See https://tools.hmcts.net/jira/browse/DMP-2367 for some reference data.